leejihee/second/과제/style.css

```
@@ -0,0 +1,33 @@
+body {
+    width: 100vw; /*화면에 보이는 최대 넓이, 높이*/
+    height: 100vh;
+    margin: 0; /*스크롤바 없애줌*/
+    background-color: coral;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.big {
+    width: 400px;
+    height: 400px;
+    background-color: ivory;
+    border: 4px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.inner {
+    width: 80px;
+    height: 80px;
+    border: 4px solid white;
+    background-color: teal;
+    margin: 15px 0;
+}
+
+#long {
+    border: 4px dashed white;
+    width: 320px;
+}
```

```suggestion
.inner {
    width: 80px;
    height: 80px;
    border: 4px solid white;
    background-color: teal;
}

#long {
    border: 4px dashed white;
    width: 320px;
    margin: 15px 0;
}
```
모든 박스에 margin을 주는 것보다는 가운데 박스에만 주는 것이 더 좋을 것 같아요~!
그리고 상하마진이 겹치는 경우는 하나만 적용되는 것도 알면 좋겠습니다!
'마진 겹침 현상'이라는 키워드로 구글링을 하면 아래 블로그 글이 나오는데 읽어보니 display: flex를 사용하는 경우 그 안에 있는 item들은 그 현상이 일어나지 않는다고 합니다. 그래서 지희님 결과물에서도 마진겹침현상이 일어나지 않아 30px씩 떨어져 있네요. 저도 새로운 사실 하나 배워갑니다😁
https://velog.io/@raram2/CSS-%EB%A7%88%EC%A7%84-%EC%83%81%EC%87%84Margin-collapsing-%EC%9B%90%EB%A6%AC-%EC%99%84%EB%B2%BD-%EC%9D%B4%ED%95%B4